astro/src/content/blog/identity-verification-before-registration.mdx

```diff
@@ -0,0 +1,296 @@
+* an advanced registration form which asks for email address, password and first name
+* a self-service registration lambda which calls the Fideo API
+* a custom error message
+
+You can log in and review this setup by [accessing the admin UI](http://localhost:9011). You should log out of the admin UI so you can easily register a user for the example application below.
+
+## Start The Example Application
+
+In order to register a new user and test out the configured functionality, you need to start a sample application. Get the example application code:
+
+```sh
+git clone https://github.com/fusionauth/fusionauth-example-identity-verification
+```
+
+Then go into the directory and install the needed packages:
+
+```sh
+cd fusionauth-example-identity-verification && npm install
+```
+
+Next, start the local server:
+
+```sh
+npm run dev
+```
+
+This starts up the sample application on your laptop. Visit [http://localhost:8080](http://localhost:8080) to make sure it is running.
+
+![A screenshot of the initial Changebank screen.](/img/blogs/identity-verification/changebank-logged-out.png)
+
+## Test The Registration Flow
+
+Next up, let's register a new user. Click the "To get started, create a new account." link.
+
+![A screenshot of the initial Changebank screen with the registration link highlighted.](/img/blogs/identity-verification/changebank-register-highlighted.png)
+
+You are going to register two times, once with a fake email address and then once with a real one.
+
+Provide a first name, email address, and password. First, try a fake address, like `test@example.com`. Such an address has a high risk value. The registration will be blocked.
+
+![A screenshot of the registration screen when registration fails.](/img/blogs/identity-verification/registration-denied.png)
+
+Then try to register with a real email address of yours, such as a `gmail.com` address. Since this is a real email address, it has a low risk value. The registration will succeed and you will be logged in.
+
+![A screenshot of the registration screen with an email address that will succeed.](/img/blogs/identity-verification/registration-will-succeed.png)
+
+Click <Breadcrumb>Logout</Breadcrumb> to log out from this account.
+
+## A Closer Look At The Validation Logic
+
+Let's take a look at the JavaScript code for the self-service registration lambda ([full code here](https://github.com/FusionAuth/fusionauth-example-kickstart/blob/main/identity-verification/lambdas/fideo.js)).
+
+Here's the validation function that Kickstart installed for you:
+
+<RemoteCode 
+   url="https://raw.githubusercontent.com/FusionAuth/fusionauth-example-kickstart/refs/heads/main/identity-verification/lambdas/fideo.js"
+   language="javascript" 
+   tags="validate" />
+
+This code does the following:
+
+* retrieves our Fideo API key from the application's configuration (where the Kickstart put it)
```

Comment:
"where the kickstart put it" is pretty vague; where _did_ the kickstart put it?

"from the application's configuration in XYZ (where the Kickstart put it)"